app/src/main/AndroidManifest.xml

```diff
@@ -56,17 +56,6 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value=".MainActivity"/>
         </activity>
-        
```

Comment:
Great clean up. Wanted to remove this unused code too. But I guess it would have been great if you'd Pull Requested all these cleanup changes in a separate PR. Next time. ;)

Now let's just move on and merge all these changes. :+1: LGTM